spec/computer_spec.rb

```diff
@@ -1,8 +1,32 @@
 require "computer"
+require "spec_helper"
+require "board"
 
 RSpec.describe Computer do
-  it "creates a new computer with a mark" do
-    computer = Computer.new("O")
+  board = Board.new(empty_board)
+  computer = Computer.new("O", board)
+
+  it "creates a new computer player with a mark" do
     expect(computer.mark).to eq("O")
   end
+  
+  it "selects an available move from the board" do
+    expect(computer.get_move.nil?).to eq(false)
```

Comment:
There should be a more expressive rspec matcher for this, something like expect(computer.get_move).to not(be_nil).

Reply:
Resolved #37